circuits/cpp/barretenberg/cpp/src/barretenberg/common/fuzzer.hpp

```diff
@@ -84,83 +84,80 @@ class FastRandom {
  *
  * @tparam T
  */
-template <typename T> concept SimpleRng = requires(T a)
-{
-    {
-        a.next()
-    }
-    ->std::convertible_to<uint32_t>;
-};
```

Comment:
Don't love how it autoformats concepts with another {} inside the {}, but that's an aside